sr-olthad/sr_olthad/framework/lms.py

```
@@ -0,0 +1,194 @@
+        self,
+        messages: list[InstructLmMessage],
+        stream_handler: LmStreamHandler | None = None,
+        **kwargs,
+        # E.g., temperature, max_tokens, top_p, presence_penalty, frequency_penalty...
+    ) -> str:
+        # Create async version of the API methods
+        async def async_chat_completion(messages, stream=False, **kwargs):
+            """Run DeepSeek's chat_completion method in a thread to make it async-compatible"""
+            loop = asyncio.get_event_loop()
+            return await loop.run_in_executor(
+                None, 
```

Ruff (0.8.2)

Trailing whitespace

(W291)